modules/schain.ts

```diff
@@ -165,8 +166,31 @@ function resolveSchainConfig(schainObject, bidder) {
   return null;
 }
 
+type SchainConfig = {
```

Comment:
```suggestion
export type SchainConfig = {
```

This way, it can be used in external systems a little easier by

```ts
import type { SchainConfig } from 'prebid.js/modules/schain';
```